server/src/main/java/org/apache/kafka/server/AssignmentsManager.java

```diff
@@ -270,8 +270,8 @@ public void run() throws Exception {
             }
             Map<TopicIdPartition, Uuid> assignment = inflight.entrySet().stream()
                     .collect(Collectors.toMap(Map.Entry::getKey, e -> e.getValue().dirId));
-            if (log.isDebugEnabled()) {
-                log.debug("Dispatching {} assignments:  {}", assignment.size(), assignment);
+            if (log.isInfoEnabled()) {
```

Comment:
Is `log.isInfoEnabled` necessary here? I think we need this log in any log level

Reply:
I adhered to the same convention as was before. It turns out the checks are useful when doing string concatenation instead of parameterised messages [0](https://www.slf4j.org/faq.html#logging_performance).

Since we're using parameterised messages, I think we can safely avoid the conditionals. Adding a commit for the same. 👍